apps/src/music/blockly/blocks/play.ts

```diff
@@ -0,0 +1,34 @@
+import BlockData from './BlockData';
+import {Block, JavaScript} from 'blockly';
+
+const BlockTypes = require('../blockTypes').BlockTypes;
+const FIELD_REST_DURATION_NAME = require('../constants')
+  .FIELD_REST_DURATION_NAME;
+
+// TODO: Move Play blocks from samples.js and simple2.js into this file.
```

Comment:
Not a super strong feeling, but while we are maintaining multiple models, I have found it useful to separate blocks into per-model files rather than per-category files.

Reply:
Fair point, I was thinking of aligning to toolbox category, but that makes sense too. In that case, maybe we can have just per-model files and then a common file or something? Mostly I want to reorganize `samples.js` into something clearer.

Reply:
I like the idea of per-model and then common, given the current state at least!